backend/src/main/java/zipgo/auth/application/KakaoOAuthClient.java

```diff
@@ -63,17 +72,28 @@ MultiValueMap<String, String> createRequestBodyWithAuthCode(String authCode) {
     }
 
     @Override
-    public OAuthResponse getMemberDetail(String accessToken) {
+    public OAuthMemberResponse getMember(String accessToken) {
+        HttpEntity<HttpHeaders> request = createRequest(accessToken);
+
+        ResponseEntity<KakaoMemberResponse> response;
+        try {
+            response = restTemplate.exchange(
+                    KAKAO_USER_INFO_URI,
+                    GET,
+                    request,
+                    KakaoMemberResponse.class
+            );
+        } catch (HttpClientErrorException e) {
+            throw new AuthException.KakaoNotFound("카카오 사용자 정보를 가져오는 중 에러가 발생했습니다");
```

Comment:
이건 저도 얼마전에 안 건데, 도움이 될 듯해 공유해요.
예외 전환을 하는 경우, 기존 예외의 정보를 포함한다면 디버깅에 유용하겠죠! 그래서 다음처럼 cause 까지 해서 RuntimeException 을 생성해주는 방법을 사용하기도 한다네용 :D
```suggestion
            throw new AuthException.KakaoNotFound("카카오 사용자 정보를 가져오는 중 에러가 발생했습니다", e);
```
```java
public KakaoNotFound(String message, Throwable cause) {
    super(message, cause);
}
```